association.go

```diff
@@ -697,6 +855,56 @@ func (a *Association) sendPayloadData(chunks []*chunkPayloadData) error {
 	return nil
 }
 
+// getPayloadDataToSend updates chunk status and returns a list of packets we can send.
+// The caller should hold the lock.
+func (a *Association) getPayloadDataToSend(onlyUnsent bool) []*packet {
```

Comment:
As I was thinking.. does it make sense to pass in the time into this so that it can be tested? Rather than relying on "Now", which makes precise testing difficult

Reply:
True. As we need to make changes around this method when we implement T3-rtx timer, let's apply that idea at that time. I will leave a note on #11.